bootstrap.d/gui-libs.yml

```diff
@@ -0,0 +1,47 @@
+packages:
+  - name: wlroots
+    source:
+      subdir: ports
+      git: 'https://gitlab.freedesktop.org/wlroots/wlroots/'
+      tag: "0.14.1"
+      version: "0.14.1"
+    tools_required:
+      - system-gcc
+      - host-libtool
+      - host-cmake
+      - wayland-scanner
+      - host-pkg-config
+      - virtual: pkgconfig-for-target
+        triple: x86_64-managarm
```

Comment:
We shouldn't hardcode any arch here, use `triple: "@OPTION:arch-triple@"` instead.